johnson_images.yaml

```diff
@@ -0,0 +1,98 @@
+entityDefinitions:
+  person:
+    fqn: "general.person"
+    entitySetName: 'JohnsonCounty_IowaCity_People3'
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "Index_ID"
+    name: "person"
+
+  image:
+    fqn: "ol.image"
+    entitySetName: "JohnsonCountyJailImages"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "Image_ID"
+      ol.imagedata:
+        type: "ol.imagedata"
+        column: "Image"
+      ol.type:
+        type: "ol.type"
+        column: "Image_Cat"
+    name: "image"
+
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "JohnsonCounty_IowaCity_JCJailStay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "Jail_Id"
+      ol.commitdatetime:
```

Comment:
This has to match the other jail yaml to correctly merge with the rest of jail data. And therefore be a different property: https://github.com/Lattice-Works/Johnson-Co-IA/blob/master/JohnsonCountyJail.yaml#L130